mappings/aeis.mapping

```diff
@@ -5,3 +5,4 @@ CLASS aeis
 	METHOD G assertTrue (ZLjava/lang/Object;)V
 	METHOD H assertTrue (ZLjava/lang/String;I)V
 	METHOD I assertTrue2 (ZLjava/lang/String;Ljava/lang/Object;)V
+	METHOD x checkArgument (ZLjava/lang/Object;)V
```

Comment:
Doesn't this throw an IllegalArgumentException, though?

Reply:
My problem with `checkArgument` is that you don't know what it actually checks for, it could be anything. Also, "checking" doesn't imply that an exception will be thrown if the check is false, while `assert` does. But to be fair, this conversation doesn't matter anyway, since this class got removed in 17.48.45.

Reply:
Then it shouldn't matter ;)
I based the name off of Preconditions.checkArgument, since that's what I thought the method was (the body lines up), but your name probably works better
~~assert shouldn't throw specifically an argument exception, though~~